chalicelib/dcc/transformer_mapping.json

```diff
@@ -7,47 +7,28 @@
       },
       {
         "source": "metadata.json",
-        "index_field": "analysis_type",
-        "dss_field": "analysis_type"
+        "index_field": "center_name",
+        "dss_field": "core_metadata.contributor"
       },
       {
         "source": "metadata.json",
-        "dss_field": "center_name",
-        "index_field": "center_name"
+        "dss_field": "project_id",
+        "index_field": "core_metadata.project"
       },
       {
-        "source": "metadata.json",
-        "dss_field": "donor_uuid",
-        "index_field": "redwoodDonorUUID"
-      },
-      {
-        "source": "metadata.json",
-        "dss_field": "project",
-        "index_field": "project"
-      },
-      {
-        "source": "metadata.json",
-        "index_field": "program",
-        "dss_field": "program"
-      },
-      {
-        "source": "metadata.json",
+        "from_file_data": true,
```

Comment:
Let's set `project`, `program` to the value of `core_metadata.project` as these values are expected to be there by the current `dcp-dashboard-service` configuration and requests to the dashboard service fail if they are not there. The project and program fields will like likely want to keep.